ntoskrnl/io/pnpmgr/devaction.c

```diff
@@ -628,11 +628,13 @@ PiCallDriverAddDevice(
     Status = IopGetRegistryValue(SubKey, REGSTR_VAL_CLASSGUID, &kvInfo);
     if (NT_SUCCESS(Status))
     {
-        if (kvInfo->Type == REG_SZ && kvInfo->DataLength > sizeof(WCHAR))
+        if ((kvInfo->Type == REG_SZ) &&
+            (kvInfo->DataLength > sizeof(UNICODE_NULL)) &&
+            (kvInfo->DataLength <= UNICODE_STRING_MAX_BYTES))
```

Comment:
Reverse this check, to add a `DPRINT1() else`? Or not wanted here?

Reply:
It's not a failure, so no need for a DPRINT